docs/guides/account-maps.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Account Maps
+---
+
+# Account Maps
+
+If you're coming from Ethereum, you must be familiar with Solidity's `mapping` data structure which allows you to map key-value pairs of different types. This mapping is stored in the smart contract's data, and is very similar to other `Map`-like data structures.
```

Comment:
We're trying to avoid any reference to Ethereum in the cookbook.

The cookbook's audience is more of new developers, so referencing another chain may cause confusion. Let's remove this reference.